src/inverseRobot.js

```diff
@@ -4,10 +4,19 @@
  * @param {object} robot
  *
  * @return {object}
-*/
+ */
 
 function inverseRobot(robot) {
-  // write code here
+  const reversedRobot = {};
+
+  for (const key in robot) {
+    if (reversedRobot.hasOwnProperty(robot[key])) {
+      return null;
+    };
+    reversedRobot[robot[key]] = key;
```

Comment:
A blank line between a block and the next command
```suggestion
    };
    
    reversedRobot[robot[key]] = key;
```